src/app/components/chat/window/output/AppChatWindowOutputItem.vue

```diff
@@ -439,13 +438,13 @@ $-min-item-width = 24px
 	display: flex
 	align-items: flex-start
 	outline: 0
-	margin-left: $left-gutter-size
+	margin-left: add-unit(v-bind(ChatWindowLeftGutterSize), px)
 	flex: auto
 	min-width: 0
 
 	@media $media-xs
 		// On small screens, reduce the left side margin to make more space for the actual messages.
-		margin-left: $avatar-size + 12px
+		margin-left: calc(v-bind(ChatWindowAvatarSize) + 12px)
```

Comment:
do we need `add-unit` here as well? this looks like it would just do  `24 + 12px` instead of `24px + 12px`